images/sonic/Dockerfile

```diff
@@ -14,8 +14,8 @@ RUN apt-get update && \
         qemu-system-x86 \
         telnet
 
-COPY --from=ghcr.io/metal-stack/mini-lab-sonic:base /sonic-vs.img /sonic-vs.img
-COPY --from=ghcr.io/metal-stack/mini-lab-sonic:base /frr-pythontools.deb /frr-pythontools.deb
+COPY --from=ghcr.io/metal-stack/mini-lab-sonic-base:master /sonic-vs.img /sonic-vs.img
+COPY --from=ghcr.io/metal-stack/mini-lab-sonic-base:master /frr-pythontools.deb /frr-pythontools.deb
```

Comment:
Doesn't the pipeline only produce images like `202411`?

Reply:
yes, I temporarily used the master tag of the image since 202311, 202411 and 202505 will be available once this PR is merged.